stable/kibana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: kibana
-version: 0.6.0
+version: 0.6.1
```

Comment:
This is a change in function, so the minor number should advance not patch number.

Reply:
Fixed!